pkgs/tools/filesystems/bcachefs-tools/default.nix

```diff
@@ -12,31 +12,20 @@
 , lz4
 , attr
 , udev
-, valgrind
 , nixosTests
 , fuse3
 , cargo
 , rustc
-, coreutils
 , rustPlatform
 , makeWrapper
 , fuseSupport ? false
+, version ? lib.importJSON ./version.json
```

Comment:
How does one generate this file? without having automation i.e. `passthru.updateScript` in place its better to use put versions in json. It will break tools like nix-update and ryantm bot.